pkgs/tools/networking/netselect/default.nix

```diff
@@ -15,11 +15,11 @@ stdenv.mkDerivation {
       --replace "-g root" "" \
       --replace "4755"    "0755"
   '';
-  
-  meta = {
-    homepage = http://alumnit.ca/~apenwarr/netselect/;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/apenwarr/netselect;
     description = "An ultrafast intelligent parallelizing binary-search implementation of \"ping\"";
-    license = "BSD";
-    platforms = stdenv.lib.platforms.linux;
+    license = licences.bsd3;
```

Comment:
s/licences/licenses/
to fix eval

Reply:
How could that happen? Typing that line should be muscle memory by now ;)